.changeset/quick-poems-know.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
+---
+
+`ActionMenu` button no longer truncates longer labels. Now follows WCAG criterion around text reflow.
```

Comment:
This change may introduce a visual breaking change in some situations. Is this something we should warn users about in the changeset?

Reply:
Thanks for checking in. In this case I would say no. 

1. Most of component changes affect the visuals in some way, so question is what's the threshold to issue a warning
1. We risk diluting the importance and impact of these warnings if we add them too frequently, so they should be used sparingly
1. I recommend adding them when they either pose an immediate risk of accessibility regressions  downstream (e.g poor contrast) **or** when design at-large will deviate considerably and we want folks to not blindly upgrade without checking.

I wouldn't see this change falling under either of those categories ☝️. The only potential risk _could_ have been the height changing, which leads to misalignment with adjacent controls but ActionMenu is not used adjacent to other form controls so we're good there. Hope that makes sense! Let me know if you have any counter considerations though @danielguillan.